include/pisa/scorer/qld.hpp

```diff
@@ -10,23 +10,26 @@ namespace pisa {
 
 template <typename Wand>
 struct qld: public index_scorer<Wand> {
-    static constexpr float mu = 1000;
-
     using index_scorer<Wand>::index_scorer;
 
+    qld(const Wand& wdata, const float mu) : index_scorer<Wand>(wdata), m_mu(mu) {}
```

Comment:
Same point about `mu`. Here, I imagine we could even explain that `mu` is a Dirichlet (is this the name of the smoothing method?) smoothing parameter.